services/blockchain-app-registry/shared/utils/fsUtils.js

```diff
@@ -71,6 +71,10 @@ const rmdir = async (directoryPath, options = { recursive: true }) => new Promis
 	);
 });
 
+const rmdir = async (directoryPath, options = { recursive: true }) => (
+	rm(directoryPath, { ...options, recursive: true })
+);
```

Comment:
We aren't awaiting the `rm` invocation. Also, could you please add a couple of tests for the `rm` method.

```suggestion
const rmdir = async (directoryPath, options) => rm(directoryPath, { ...options, recursive: true });
```
or
```suggestion
const rmdir = async (directoryPath, options) => rm(
	directoryPath,
	{ ...options, recursive: true },
);
```

Reply:
The returned promise was from `rmdir` but the promise from `rm` wasn't awaited since that promise wasn't returned by `rmdir`.